src/System.Private.Xml/tests/XmlReader/Tests/AsyncReaderLateInitTests.cs

```diff
@@ -68,6 +68,7 @@ public static void ReadAfterInitializationWithTextReaderOnAsyncReaderDoesNotThro
 
         [Fact]
         [Trait("a", "b")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetcoreUwp)]  // Path cannot be resolved in UWP
```

Comment:
Should we load an active issue on this tests for those platforms? cc: @Chrisboh since we recently talked about how should we disable tests on specific frameworks.